person.rb

```diff
@@ -1,21 +1,25 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_accessor :age, :name
   attr_reader :id
 
   def initialize(_id, age, parent_permission: true, name: 'unknown')
+    super(name)
     @id = Random.rand(1..1000)
     @age = age
     @parent_permission = parent_permission
-    @name = name
   end
```

Comment:
- [ ] While trying to run your application, I got below error message complaining about the wrong number of arguments in the `initialize` method. Kindly look into it and fix.

### Error
![error](https://github.com/Metaverse-Nabeel/OOP-school-library/assets/26597432/ef0705e7-249c-4859-b4d1-bd2e5eaaaad6)